src/client/datascience/jupyter/kernels/kernelSelections.ts

```diff
@@ -24,6 +24,7 @@ import { IKernelSelectionListProvider, IKernelSpecQuickPickItem, LiveKernelModel
  * Given a kernel spec, this will return a quick pick item with appropriate display names and the like.
  *
  * @param {IJupyterKernelSpec} kernelSpec
+ * @param pathUtils
```

Comment:
```suggestion
 * @param {IPathUtils} pathUtils
```

or remove `IPathUtils` from `getQuickPickItemForActiveKernel`